validation-test/stdlib/Arrays.swift.gyb

```diff
@@ -138,12 +144,435 @@ ArrayTestSuite.test("${Kind}/popLast") {
 }
 % end
 
-public struct ArrayIndexTest<T: Collection> {
-  public enum RangeBox {
-    case closed(ClosedRange<Int>)
-    case halfClosed(Range<Int>)
+//===----------------------------------------------------------------------===//
+// COW(🐄) tests
+//===----------------------------------------------------------------------===//
+
+class COWBox<T: Equatable & CustomStringConvertible>: Equatable, CustomStringConvertible {
```

Comment:
Style nitpick: stdlib codebase is 80 columns wide and `:` has spaces on the side where the type is, so:

```swift
class COWBox <
  T : Equatable & CustomStringConvertible
> : Equatable, CustomStringConvertible {
```